rpa-home/modules/ROOT/pages/automation-userrolesandpermissions-privilegesbymodule.adoc

```diff
@@ -0,0 +1,358 @@
+
+[cols="1,2"]
+|===
+|*Privilege* |*Permitted Action*
+
+|*Process Monitoring Open*
+|Open all views of the *Process Monitoring* module
+
+|*Robot State and Operation Open*
+|Open the *Robot State and Operation* view
+
+|*Process Streaming Open*
+|Open the *Process Streaming* view
+
+|===
+
+
+
+
+== Privileges of the *My RPA* module
+
+[cols="1,2"]
+|===
+|*Privilege* |*Permitted Action*
+
+|*Global Process Evaluation View*
+|Authorization to view all processes in the views *Process Evaluation* and *My RPA* &gt; *Team Backlog*, regardless of whether or not the current user belongs to the Process Team.
+
+|*Unprocessed Task List Open*
+|Open the *Unprocessed Tasks* view.
+
+|*Unprocessed Task List Edit*
+|Change the *Unprocessed Tasks* view.
+
+|===
+
+
+
+
+== Privileges of the *Dashboard* module
+
+[cols="1,2"]
+|===
+|*Privilege* |*Permitted Action*
+
+|*Dashboard Open*
+|Open all views of the *Dashboard* module
+
+|*Dashboard Administration*
+|Create, change, and delete all objects of the *Dashboard* module
+
+|===
+
+
+
+
+== Privileges of the *Alerting* module
+
+[cols="1,2"]
+|===
+|*Privilege* |*Permitted Action*
+
+|*Alerting Open*
+|Open all views of the *Alerting* module
+
+|*Alerting Administration*
+|Create, change, and delete all objects of the *Alerting* module
+
+|===
+
+
+
+
+== Privileges of the *RPA Builder* category
+
+The *Activity Library* is managed exclusively in the RPA Builder.
+
+Learn how to xref:rpa-builder::ui-overview.adoc#explorer-pane[find the Activity Library in RPA Builder].
+
+[cols="1,2"]
+|===
+|*Privilege* |*Permitted Action*
+
+|*Activity Library Administration*
+|Authorization to manage the *Activity Library* in the RPA Builder.
+
+|*Activity Library Open*
+|Authorization to open the *Activity Library* in the *X1 Design Studio*.
+
+|===
+
+
+
+
+// Future privileges
+//
+// == Privileges of the *RPA Public REST API* category
+//
+// You will require the special privileges listed below to use the *RPA Public REST API*.
+//
+// The individual Privileges you'll need to call can be found in the  *RPA Public REST API* documentation.
+//
+// [cols="1,2"]
+// |===
+// |*Privilege* |*Permitted Action*
+//
+// |*REST API Application*
+// |Administration of Process Applications using the *RPA Public REST API*
+//
+// |*REST API Global Variables/Credential Pool*
+// |Administration of Global Variables and the Credential Pool using the *RPA Public REST API*
+//
+// |*REST API Processes*
+// |Administration of Processes using the *RPA Public REST API*
+//
+// |*REST API Process Board*
+// |Administration of Process Boards using the *RPA Public REST API*
+//
+// |*REST API Production Configurations*
+// |Administration of Process Configurations using the *RPA Public REST API*
+//
+// |*REST API Robots*
+// |Administration of *X1 Bots* using the *RPA Public REST API*
+//
+// |*REST API Test Configurations*
+// |Administration of Test Configurations using the *RPA Public REST API*
+//
+// |===
```

Comment:
These currently have no function as we do not have a public REST API and will be removed soon.

Reply:
Also X1 probably shouldn't be mentioned

Reply:
Was already commented out for future use, but is removed now

Reply:
Ah my bad just read it and did not take into account that this is commented out.